README.adoc

```diff
@@ -14,22 +14,30 @@ across servers, tiers, on-premises and in the cloud. It will also enable detaile
 performance analysis to be performed of the individual components that make up an
 application.
 
-== Build
+== Build & Run
 
-To build the project, run:
+To build and run execute:
 
 ```shell
-mvn clean install [ -Ptests ]
+$ mvn clean install -Pdev -DskipTests [-Ptests]
+$ dist/target/hawkular-apm-${version}/bin/standalone.sh [-Djboss.socket.binding.port-offset=1331]
 ```
+For login use jdoe:password
 
-The optional 'tests' profile will automatically run integration tests.
-
-The distribution (a preconfigured version of WildFly with APM installed) can be
-found here: _dist/target/hawkular-app-dist-<version>.zip_.
+ * `Djboss.socket.binding.port-offset=1331` - deploy application on http://zipkin.io/[Zipkin] 9411 port. Standard
+ APM port is 8080.
+ * `dev` - create user jdoe:password
```

Comment:
For this, and the next line, it might be worth putting the whole profile * '-Pdev'